tests/ui/check-cfg/target_feature.rs

```diff
@@ -0,0 +1,13 @@
+// This test makes sure that we don't emit a long list of possible values
+// but that we stop at a fix point and say "and X more".
```

Comment:
Question: is this comment accurate? AFAICT the corresponding stderr does emit all possible values, right?

Reply:
Mixed up with the other test `and_more_diagnostic.rs` which does limit the output and prints `and X more`.